openapi_tester/schema_tester.py

```diff
@@ -123,7 +158,7 @@ def _get_status_code(schema: dict, status_code: Union[str, int], error_addon: st
         """
         if str(status_code) in schema:
             return schema[str(status_code)]
-        elif int(status_code) in schema:
+        if int(status_code) in schema:
```

Comment:
pylint, doesnt like redundant elif not required due to return.

Reply:
i dont particularly care about such stuff - for me linters are meant to make sure we do not need to worry about style, because of opinions. But if you feel this is important you can disable the pylint rule and change the stuff.